cypress/integration/common/deal-wizard.ts

```diff
@@ -0,0 +1,91 @@
+import { Then, When } from "@badeball/cypress-cucumber-preprocessor/methods";
+
+Then("I am presented the option to choose a partner", () => {
+  cy.url().should("match", /(initiate\/token-swap$)/)
+})
```

Comment:
This only checks the url, but should also verify "the option to choose a partner"

Reply:
There is already a scenario that covers that in choose-deal-type.feature:

```
  Scenario: Verify correct Token Swap types
    Given I select Token Swap
    Then I can see Open Proposal and Partnered Deal
```

Is there a need to double that?

Reply:
In that case what is the difference between
1. Then I am presented the option to choose a partner
2. Then I can see Open Proposal and Partnered Deal
?

All I'm trying to say is, that based on what I read "I am presented the option to choose a partner", I expected to test to verify for the option, because it is expecially the `Then` step.

I do see a challenge to differentiate on what to put into Scenario, and what to put into Steps,
and can't really give recommendations at the top of my head.

If you feel, the Steps and Scenarios are fine as is, then fine with me.
We can refactor later if ever confusion comes up.

Reply:
I am presented the option to choose a partner - This is just checking the URL
I can see Open Proposal and Partnered Deal - This is checking the UI

I'm not sure what's the correct way to handle this ;) 
It's a good idea to leave it as it is and dedicate some time for refactoring later on